app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb

```diff
@@ -12,10 +12,30 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Credential < Mana
 
   FRIENDLY_NAME = "Ansible Automation Inside Credential".freeze
 
+  include ManageIQ::Providers::EmbeddedAnsible::CrudCommon
+
   def self.provider_params(params)
     super.merge(:organization => ManageIQ::Providers::EmbeddedAnsible::AutomationManager.first.provider.default_organization)
   end
 
+  def self.params_to_attributes(_manager, params)
+    obj_attributes = {:options => {}}
+
+    params.except(:id).each do |key, value|
+      if column_names.include?(key.to_s)
+        obj_attributes[key] = value
+      else
+        obj_attributes[:options][key] = value
```

Comment:
I think we still need to deal with whichever of these may need to be encrypted. Maybe we should have some kind of list in the sub classes?

Reply:
Possibly, though I figured this is something that would be overwritten in the subclasses as needed.

I kinda figure that `column_names` is going to be a bit rudimentary and will need to change over time, and I figured we would make use of `alias_attribute` as needed to apply credential type differences to the schema of the `authentications` table.

Reply:
Yeah, I'm not sure this implementation `column_names.include?` will ever be able to be used directly. I'm thinking we should ensure this is implemented in all of the sub classes and not create a base implementation here at all.